src/tracer.js

```diff
@@ -18,14 +19,29 @@ export default class Tracer {
     /**
      * Starts and returns a new Span representing a logical unit of work.
      *
-     * @param  {string|object} nameOrFields - if the given argument is a
+     * For example:
+     *
+     *     // Start a new (parentless) root Span:
+     *     var parent = Tracer.startSpan('DoWork');
+     *
+     *     // Start a new (child) Span:
+     *     var child = Tracer.startSpan('Subroutine', {
+     *         reference: Tracer.childOf(parent.context()),
```

Comment:
Is using `parent` and a span object directly no longer allowed?

This is a little concerning to me as it adds complexity to the simple case (where I'm new to tracing and am not yet ready for / care about the finer distinctions). In broad terms, I'd argue "mostly everyone" gets what a parent-child relationship is, so `Tracer.startSpan('Subroutine', { parent : parentSpan })` is pretty straightforward.  

By contrast, the snippet above requires two calls to `Tracer`, a sort of redundancy isn both a `childOf` call and a `parent` object, and the next concept of a `context()` (which before I could get away with not knowing about).

I'd argue rather strongly the `parent : <parentObject>` call should still be allowed if that is not a violation of the OpenTracing spec.